### PR TITLE
Pey/2008 flights 7 m

### PR DESCRIPTION
**Pull request by psilvest**
Commit to add flights dataset.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+2008_Flights_7M/dataset/.DS_Store
```
Comment (andrewseidl) [inline .gitignore:2]: Should probably make this just `.DS_Store` so it's applied everywhere.

File: 2008_Flights_7M/omnisci_ddl/flights_2008_7M.sql [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  dest_country TEXT ENCODING DICT(32),
+  dest_lat FLOAT,
+  dest_lon FLOAT,
+  origin_merc_x FLOAT,
+  origin_merc_y FLOAT,
+  dest_merc_x FLOAT,
+  dest_merc_y FLOAT)
+WITH (FRAGMENT_SIZE=2000000);
+
+copy flights_2008_7M from 's3://omnisci-community/datasets/airline_on-time_performance/flights_2008_7M.csv.gz' with (header='true');
```
Comment (andrewseidl) [inline 2008_Flights_7M/omnisci_ddl/flights_2008_7M.sql]: Missing newline at end of file, not sure if omnisql will always accept that.

File: 2008_Flights_7M/README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# USA Flights
+
+![flights dashboard](https://github.com/omnisci/community_datasets/blob/master/USAflights/dashboard_img/usa_flights)
```
Comment (andrewseidl) [inline 2008_Flights_7M/README.md]: This link is incorrect

**Review by andrewseidl — CHANGES_REQUESTED** [review]
Directory name could be better, maybe something like `airline-on-time-performance`?

File: 2008_Flights_7M/omnisci_ddl/flights_2008_7M.sql (hunk 2 above)
Comment (psilvest) [reply 2008_Flights_7M/omnisci_ddl/flights_2008_7M.sql]: added newline at the end.

**Comment by psilvest** [comment]
> Directory name could be better, maybe something like `airline-on-time-performance`?

Changed to suggested

File: .gitignore (hunk 1 above)
Comment (psilvest) [reply .gitignore:2]: Done.

File: 2008_Flights_7M/README.md (hunk 3 above)
Comment (psilvest) [reply 2008_Flights_7M/README.md]: Fixed.

**Review by andrewseidl — APPROVED** [review]

**Pull request merged**